pyproject.toml

```diff
@@ -30,12 +30,12 @@ dependencies = [
   # changing the version
   # (plugins.mkdocstrings.handlers.python.import)
   "frequenz-client-microgrid >= 0.18.0, < 0.19.0",
+  "frequenz-microgrid-component-graph >= 0.2.0, < 0.3",
```

Comment:
Ideally I would like to add only dependencies that are at `> 1.0.0` in the SDK. The v0.x approach proved to be confusing and error prone. Can we make a 1.0 release for `frequenz-microgrid-component-graph` soon? I guess this is only expected to be used inside the SDK, so it is not that bad, but still. I think we can still do a lot of incremental improvements with backwards compatible changes, and at some point it is fine to bump to v2.0.0 if we need to do some cleanup.

Reply:
We can, will look into it for the next release.